node_modules/oae-content/lib/authz.js

```diff
@@ -39,6 +39,25 @@ var canManageContent = module.exports.canManageContent = function(ctx, content,
 };
 
 /**
+ * Determine if the user invoking the current request is allowed to edit a given content item
+ *
+ * @param  {Context}        ctx                     Standard context object containing the current user and the current tenant
+ * @param  {Content}        content                 The content object we want to check
+ * @param  {Function}       callback                Standard callback function
+ * @param  {Object}         callback.err            An error that occurred, if any
+ * @param  {Boolean}        callback.canManage      Whether or not the user can edit the content item
+ */
+var canEditContent = module.exports.canEditContent = function(ctx, content, callback) {
+    if (!ctx.user()) {
+        return callback(null, false);
+    } else if (ctx.user().isAdmin(content.tenant.alias)) {
+        return callback(null, true);
+    }
+
+    return AuthzAPI.hasRole(ctx.user().id, content.id, ContentConstants.roles.EDITOR, callback);
```

Comment:
Can we do AuthzAPI.getAllRoles and return `true` if the user is either an editor or manager?